### PR TITLE
Updated README.md and LICENSE

### PR DESCRIPTION
**Pull request by eierina**
Added README.md and Apache License references

### CODE REVIEW

**Review by r3domfox — APPROVED** [review]
LGTM

**Review by MarkOldfield — APPROVED** [review]

File: src/r3/atomic-swap/corda/README.md [hunk 1]
```diff
@@ -1 +1,91 @@
...
+
+## Development Status
+
+The atomic swap reference code is currently under development but nearing completion. Some of its components are in the process of refinement and preparation for community access.
+
+### Component Overview and Status
+
+The following is a list of components included in this project. All of these are subject to future changes and are currently under development.
+
+1. **EVM Interoperability Service**: This service enables flows to execute asynchronous EVM transactions and calls, waiting for the result. It is fully implemented and tested.
```
Comment (gendal) [inline src/r3/atomic-swap/corda/README.md:15]: is there a mapping from these components to specific files or subdirectories?  If so, include a link so the reader can navigate to the code in question?

File: src/r3/atomic-swap/corda/README.md [hunk 2]
```diff
@@ -1 +1,91 @@
...
+
+4. **Atomic Swap Flows**: Basic atomic swap flows for executing Corda-EVM DvP and PvP scenarios are under active development.
+
+5. **Event Subscription Service**: This component is responsible for subscribing to and handling events. It is currently under active development and nearing completion. Some features are experimental and will be moved to a separate external module in future iterations.
+
+#### Coming Soon
+
+6. **Patricia Merkle Trie Component**: This upcoming component will support EVM events and transaction validation, production of events and transaction proofs, and verification of proofs.
+
+7. **Full Atomic Swap Flows**: Full atomic swap flows for executing Corda-EVM DvP and PvP scenarios are in the pipeline and will be implemented in the near future.
```
Comment (gendal) [inline src/r3/atomic-swap/corda/README.md:29]: what is the difference between 'basic' flows (4 above) and 'full'?

File: src/r3/atomic-swap/corda/README.md [hunk 3]
```diff
@@ -1 +1,91 @@
...
+```
+or
+```
+./gradlew build 
+```
+if you have manually prepared the EVM test environment.
+
+#### Deploy
+
+Plain and Dockerized deploymet is under development
```
Comment (gendal) [inline src/r3/atomic-swap/corda/README.md:69]: deployment

File: src/r3/atomic-swap/corda/README.md [hunk 4]
```diff
@@ -1 +1,91 @@
...
+
+To set up the test environment proceed as follows:
+- change directory to samples/testnet and open two terminals to that directory
+- on the first terminal run `npm install` and wait for the required packages to be installed - this step is required once.
+- again on the first terminal run `npx hardhat node` - it will print a number of accounts and will start printing block numbers in the form `Mined empty block range #m to #n`
+- on the second terminal, once the first the hardhat node is running, enter `npx hardhat run deploy.js --network localhost` and wait for the shell prompt to return (without errors)
+
+If you followed the steps above correctly, on the second terminal you will see the following output:
+
+Gold Tethered (GLDT) Token deployed to: 0x5FbDB2315678afecb367f032d93F642f64180aa3</br>
```
Comment (gendal) [inline src/r3/atomic-swap/corda/README.md:83]: <br>?

**Review by gendal — COMMENTED** [review]
Thanks Edoardo. Wow - you have far more here than I realised!  Four minor comments.

**Pull request merged**